app/components/geoblacklight/location_leaflet_map_component.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Geoblacklight
+  class LocationLeafletMapComponent < ViewComponent::Base
+    def initialize(id: "map", label: nil, data_map: "", map_geometry: nil)
+      @id = id
+      @label = label
+      @data_map = data_map
+      @map_geometry = map_geometry || Settings.HOMEPAGE_MAP_GEOM
+      super
+    end
+
+    def before_render
```

Comment:
If you are hardcoding `id: "map"` does it need to be in the init arguments?
Same with label -- if label is always nil and then overwritten during `before_render` to pull from the template file, does it need to be in the arguments?

Reply:
That is definitely a possibility since for the two tickets I have used label and id are the same. They seemed like fields that would be ones you might want to have a different label for on different pages and the id is one I could see needing to be changable due to how leaflet works. If you wanted to put multiple leaflets on the page, being able to change those would be preferable. I can take it out for now, but I thought it would probably be good to no longer hardcode these fields.

Reply:
Sounds good, just curious!